main.bundle.js

```diff
@@ -52,19 +52,48 @@
 
 	var _currentWeather2 = _interopRequireDefault(_currentWeather);
 
+	var _backgroundImage = __webpack_require__(6);
+
+	var _backgroundImage2 = _interopRequireDefault(_backgroundImage);
+
 	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
-	// This file is in the entry point in your webpack config.
-	$('form').on('submit', function (event) {
+	$(document).ready(function () {
+	  $('#location-form').on('submit', getLocationInfo);
+	  $('#sign-up').on('submit', registerUser);
+	}); // This file is in the entry point in your webpack config.
+
+
+	var registerUser = function registerUser(event) {
 	  event.preventDefault();
-	  var location = $('input').val().toLowerCase();
-	  $.ajax({
-	    type: "GET",
-	    url: "http://localhost:3000/api/v1/forecast?location=" + location
-	  }).then(setCurrentWeather).catch(function (error) {
-	    console.log('Error: ' + error);
+	  $('.errors').text('');
+	  var email = $('input')[0].value;
+	  var password = $('input')[1].value;
+	  var confPassword = $('input')[2].value;
+	  if (!password || !confPassword || password !== confPassword) {
+	    displayError("Error: Please check passwords");
+	  } else {
+	    createUser(email, password, confPassword);
+	  };
+	};
+
+	var createUser = function createUser(email, password, confPassword) {
+	  var usersUrl = ("http://localhost:3000/api/v1") + "/users";
+	  $.post(usersUrl, { email: email, password: password, password_confirmation: confPassword }).done(function (data) {
+	    document.cookie = 'api_key=' + data.api_key;
```

Comment:
I've actually never used document.cookie before! In looking at the docs, this looks like the perfect use case. 

We have primarily been using [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) to save this type of information... but we also haven't been working with logins very much since we are just learning how to build a backend now. 

I wouldn't recommend using localStorage though, your method seems more secure. I'm definitely going to look into adding it on my BikeSee project since I have a whole extra week to work in it now 😃!

Reply:
🚲 🍪 😃